src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -842,6 +978,39 @@ namespace OpenLoco::GameCommands
         return doCommand(GameCommand::raiseLand, regs);
     }
 
+    struct RaiseLandArgs
+    {
+        static constexpr auto command = GameCommand::raiseLand;
+
+        RaiseLandArgs() = default;
+        explicit RaiseLandArgs(const registers& regs)
+            : centre(World::Pos2(regs.ax, regs.cx))
+            , pointA(World::Pos2(regs.edx, regs.ebp))             // lower bits
+            , pointB(World::Pos2(regs.edx >> 16, regs.ebp >> 16)) // higher bits
+            , corner(regs.di)
+            , flags(regs.bl)
```

Comment:
don't add flags here.

Reply:
Sorry, how I should do then? Just remove or move to somewhere else?

Reply:
Ahhh, sorry. Just remembered that the point is just to remove these.